src/types/timezone.jl

```diff
@@ -1,4 +1,28 @@
-const TIME_ZONE_CACHE = Dict{String,Tuple{TimeZone,Class}}()
+# Thread-local TimeZone cache, which caches time zones _per thread_, allowing thread-safe
+# caching. Note that this means the cache will grow in size, and may store redundant objects
+# accross multiple threads, but this extra space usage allows for fast, lock-free access
+# to the cache, while still being thread-safe.
+const THREAD_TZ_CACHES = Dict{String,Tuple{TimeZone,Class}}[]
```

Comment:
Mainly for me, I just want to make it very obvious this now a vector:
```suggestion
const THREAD_TZ_CACHES = Vector{Dict{String,Tuple{TimeZone,Class}}}()
```

Reply:
Awesome, yeah, thanks 👍